docs/en/sql-reference/statements/alter/ttl.md

```diff
@@ -82,4 +82,4 @@ The `TTL` is no longer there, so the second row is not deleted:
 ### See Also
```

Comment:
```suggestion
**See Also**
```

Reply:
Why need these meaningless changes?

Reply:
Specifically, this is not necessary. In some places, there are anchors for "see also" - they get in the way. We specified this in the templates.